docs/advanced.rst

```diff
@@ -740,7 +740,7 @@ A ``system.conf`` could look like this:
 
 .. _sec-mbr-partition:
 
-Update Boot Partitions in MBR
+Update Boot Partition in MBR
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
If removing the plural 's' in the line above then one '~' should be removed here as well.